src/components/learner-credit-management/LearnerCreditGroupMembersTableAction.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+import { Download } from '@edx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+import snakeCase from 'lodash/snakeCase';
+import { saveAs } from 'file-saver';
+import EnterpriseAccessApiService from '../../data/services/EnterpriseAccessApiService';
+import {
+  useBudgetId,
+  useSubsidyAccessPolicy,
+} from './data';
+
+const LearnerCreditGroupMembersTableAction = ({
+  tableInstance,
+}) => {
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const groupId = subsidyAccessPolicy.groupAssociations[0];
+
+  const getCsvFileName = () => {
+    const titleNoWhitespace = subsidyAccessPolicy.displayName.replace(/\s+/g, '');
+    const currentDate = new Date();
+    const year = currentDate.getUTCFullYear();
+    const month = currentDate.getUTCMonth() + 1;
+    const day = currentDate.getUTCDate();
+    return `${titleNoWhitespace}-${year}-${month}-${day}.csv`;
+  };
+
+  return (
+    <Button
+      onClick={() => {
+        const options = {
+          format_csv: true,
+          traverse_pagination: true,
+          group_uuid: groupId,
+        };
+        if (tableInstance.state.sortBy[0]) {
+          options.sort_by = snakeCase(tableInstance.state.sortBy[0].id);
+          if (!tableInstance.state.sortBy[0].desc) {
+            options.is_reversed = !tableInstance.state.sortBy[0].desc;
+          }
+        }
+        if (tableInstance.state.filters[0]) {
+          options.user_query = snakeCase(tableInstance.state.filters[0].value);
+        }
+        EnterpriseAccessApiService.fetchSubsidyHydratedGroupMembersData(
+          subsidyAccessPolicyId,
+          options,
+        ).then(response => {
+          // download CSV
+          const blob = new Blob([response.data], {
+            type: 'text/csv',
+          });
+          saveAs(blob, getCsvFileName());
+        }).catch(err => {
+          logError(err);
+          // TODO: what should the UX be for error here?
```

Comment:
Have we gotten an answer for this question?

Reply:
done